packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx

```diff
@@ -62,31 +77,36 @@ export const MeetingCallControlBar = (props: MeetingCallControlBarProps): JSX.El
       horizontal
       className={mergeStyles(temporaryMeetingControlBarStyles, callControlsContainerStyles, controlBarContainerStyles)}
     >
-      <Stack.Item grow>
-        <CallAdapterProvider adapter={props.callAdapter}>
-          <CallControls options={callControlsOptions} increaseFlyoutItemSize={props.mobileView} />
-        </CallAdapterProvider>
-      </Stack.Item>
-      <Stack.Item>
-        {isEnabled(props.meetingCallControlOptions?.chatButton) !== false && (
-          <ChatButton
-            checked={props.chatButtonChecked}
-            showLabel={true}
-            onClick={props.onChatButtonClicked}
-            data-ui-id="meeting-composite-chat-button"
-            disabled={props.disableButtonsForLobbyPage}
-          />
-        )}
-        {isEnabled(props.meetingCallControlOptions?.peopleButton) !== false && (
-          <PeopleButton
-            checked={props.peopleButtonChecked}
-            showLabel={true}
-            onClick={props.onPeopleButtonClicked}
-            data-ui-id="meeting-composite-people-button"
-            disabled={props.disableButtonsForLobbyPage}
-          />
-        )}
-      </Stack.Item>
+      {callControlsOptions !== false && (
```

Comment:
Since `CallControls` now returns `<></>` when options are `false`, this check should no longer be required (once you allow `boolean` in the props again).